docs/releasing.md

```diff
@@ -1,8 +1,37 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
+
+### Release manager
+
+Rather than a rotation, release manager is appointed. However if the appointed manager is absent, the next engineer in the following list owns it:
+
+    bckenny, paulirish, patrickhulce
+
+Release manager follows the below _Release Process_.
+
+### Release publicity
+
+1. Release mgr copies changelog to a new [Releases](https://github.com/GoogleChrome/lighthouse/releases). Tags and ships it.
+1. Release mgr adds Release notes to the repo's [changelog](https://github.com/GoogleChrome/lighthouse/blob/master/changelog.md).
```

Comment:
do we intend to split this out from the PR with the version bumps? maybe worth noting to do it in the same if not :)

Reply:
oops. left this line in by mistake. 

no changelog is done with that PR.   and then reused here.